package.json

```diff
@@ -40,18 +40,18 @@
     "pa11y-reporter-json": "^1.0.0",
     "pa11y-runner-axe": "^1.0.1",
     "pa11y-runner-htmlcs": "^1.2.0",
-    "puppeteer": "^1.13.0",
-    "semver": "^5.6.0"
+    "puppeteer": "^1.20.0",
+    "semver": "^5.7.1"
   },
   "devDependencies": {
     "@rowanmanning/make": "^1.0.1",
-    "eslint": "^6.0.1",
-    "mocha": "^6.1.4",
+    "eslint": "^6.8.0",
+    "mocha": "^6.2.2",
     "mockery": "^2.0.0",
-    "nyc": "~13.2.0",
+    "nyc": "^15.0.0",
```

Comment:
Can you leave the `nyc` dependency as is. There's a bug where it doesn't generate coverage results when we bumped the version up. @josebolos might remember what was up with that?

Reply:
I've inspected the output of `make clean && npm i && make ci` on both master and my branch.  Both show
```
  253 passing (23s)

-------------|---------|----------|---------|---------|-------------------
File         | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
-------------|---------|----------|---------|---------|-------------------
All files    |     100 |      100 |     100 |     100 |                   
 action.js   |     100 |      100 |     100 |     100 |                   
 pa11y.js    |     100 |      100 |     100 |     100 |                   
 reporter.js |     100 |      100 |     100 |     100 |                   
 runner.js   |     100 |      100 |     100 |     100 |                   
-------------|---------|----------|---------|---------|-------------------
```
Additionally, coverage files produced in `.nyc_output/*json` by both branches are 56378 bytes (on my workstation) and whose contents are identical except for the hash values for the keys '_coverageSchema', 'contentHash', and 'hash'.  Perhaps the coverage bug is no longer present?  The reports from this PR build compared to previously-merged PRs appear superficially similar on Travis as well.

Reply:
If I'm not mistaken, it should have been fixed on 15.0.0, but I haven't had the time to check if that's the case.

Reply:
And from the build server:
```
-------------|---------|----------|---------|---------|-------------------
File         | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
-------------|---------|----------|---------|---------|-------------------
All files    |     100 |      100 |     100 |     100 |                   
 action.js   |     100 |      100 |     100 |     100 |                   
 pa11y.js    |     100 |      100 |     100 |     100 |                   
 reporter.js |     100 |      100 |     100 |     100 |                   
 runner.js   |     100 |      100 |     100 |     100 |                   
-------------|---------|----------|---------|---------|-------------------
```

great, looks like that's fixed. sorry, must have borked looking at the build server logs